.changeset/perfect-crews-speak.md

```diff
@@ -0,0 +1,5 @@
+---
+"saleor-dashboard": patch
+---
+
+Searching for countries and other items is now more efficient, making it easier to find what you need. Additionally, the Dashboard Navigator UI has been improved to match the rest of the application, providing a more consistent experience.
```

Comment:
what does mean "improved", "consistent", "easier to find" - how does that impact the user? If this is just migration, meaning technical stuff / maintenance, you can prepare more developer-related descriptions.

_The "fuzzaldrin" is no longer used within the codebase, this means we have migrated to "Fuse.js"_

Reply:
> what does mean "improved", "consistent", "easier to find"
I'm not sure I understand the confusion. From the user perspective search now works better and gives relevant results. I mentioned country search specifically as the link PR from community was fixing that particular search. 

> If this is just migration, meaning technical stuff / maintenance, you can prepare more developer-related descriptions.
According to our changeset guide we have to be user-focused and focused on user workflow. Why does a user have to care about a migration of a library? I wrote what user can expect when this change is merged and released.